build.gradle

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 group = 'com.github.ben-manes'
-version = '0.39.0'
+version = '0.39.1-SNAPSHOT'
```

Comment:
Just noticed this - should we revert this since I guess it's for debugging?

Reply:
yeah, this can be incremented to `0.40.0` for the merge and release.